src/pages/public/about-us-page/intro-section/IntroSection.tsx

```diff
@@ -7,17 +7,15 @@ export const AboutUsIntro = () => {
         <div className="about-us-block">
             <img src={background} className="background-img" alt="Men and Horse" />
             <img src={background} className="color-overlay" alt="Men and Horse" />
-            <h1 className="about-us-main-title">
-                <span className="highlighted">{ABOUT_US_DATA.INTRO_TITLE.FIRST_HIGHLIGHT}</span>
-                {ABOUT_US_DATA.INTRO_TITLE.MIDDLE_PART}
-                <span className="highlighted">{ABOUT_US_DATA.INTRO_TITLE.SECOND_HIGHLIGHT}</span>
-            </h1>
-            <div className="title-details">
-                <p>{ABOUT_US_DATA.INTRO_DETAILS.FIRST_LINE}</p>
-                <p>{ABOUT_US_DATA.INTRO_DETAILS.SECOND_LINE}</p>
-                <p>{ABOUT_US_DATA.INTRO_DETAILS.THIRD_LINE}</p>
-                <p className="paragraph-break">{ABOUT_US_DATA.INTRO_DETAILS.FOURTH_LINE}</p>
-                <p>{ABOUT_US_DATA.INTRO_DETAILS.FIFTH_LINE}</p>
+            <div className="about-us-info">
+                <h1 className="about-us-main-title">
+                    <span className="highlighted">{ABOUT_US_DATA.INTRO_TITLE.FIRST_HIGHLIGHT}</span>
+                    {ABOUT_US_DATA.INTRO_TITLE.MIDDLE_PART}
```

Comment:
maybe CONTENT word is a little bit better :)
```suggestion
                    {ABOUT_US_DATA.INTRO_TITLE.CONTENT}
```